templates/layout/base.html

```diff
@@ -14,9 +14,16 @@
             {{!-- Change document class from no-js to js so we can detect this in css --}}
             document.documentElement.className = document.documentElement.className.replace('no-js', 'js');
         </script>
+        {{!-- Load Lazysizes script ASAP so images will appear --}}
+        <script>
+            {{!-- Only load visible elements until the onload event fires, after which preload nearby elements. --}}
+            window.lazySizesConfig = window.lazySizesConfig || {};
+            window.lazySizesConfig.loadMode = 1;
+        </script>
+        <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js'}}"></script>
```

Comment:
Whoa, javascript in the head?!?!

Yes - we need this to make sure images actually show up on the page, so this few KB file is mission-critical.

I've marked it as `async` so it won't block render, but we want it to execute as soon as it has downloaded.